src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -131,10 +131,10 @@
   </Node>
 
   <!--
-  This node is used to represent an expression of a certain type, when attempting to bind its Index or Range pattern indexer
+  This node is used to represent an expression of a certain type, when attempting to bind its Index or Range fallback indexer
   It does survive past initial binding but will be replaced with a synthesized argument in DAG lowering.
   -->
-  <Node Name="BoundIndexOrRangeIndexerPatternValuePlaceholder" Base="BoundValuePlaceholderBase">
+  <Node Name="BoundIndexOrRangeIndexerFallbackValuePlaceholder" Base="BoundValuePlaceholderBase">
```

Comment:
>Fallback [](http://example.com/codeflow?start=38&length=8)

What is the "Fallback" supposed to convey. Do we need to have it in the name?

Reply:
"Fallback" is how we've been referring to such members in LDM. This is the fallback when `this[Index]` or `this[Range]` don't exist.
Looking at the [ranges spec](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-8.0/ranges.md#implicit-index-support) I'd also be fine with "ImplicitIndexer".

Reply:
I think "ImplicitIndexer" is better. Also, the "ValuePlaceholder" feels confusing. Does the node represent an argument for the indexer or the value of the indexer (i.e. result of its evaluation)?